corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -554,6 +558,14 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             // Reconcile state of "select all" checkbox
             self.ui.selectAllCheckbox.prop("checked", !_.difference(self._allCaseIds(), self.selectedCaseIds).length);
         },
+
+        verifySelectedCaseIdsLessThanMaxSelectValue: function () {
+            if (this.selectedCaseIds.length > this.maxSelectValue) {
+                let errorMessage = _.template(gettext("You have selected more than the maximum selection limit of "
```

Comment:
This doesn't quite work - the overall idea is that whatever is passed to `gettext` needs to be a static string, so maxSelectValue needs to be a template variables, something like `_.template(gettext("You have selected more than <%= value %> values please fix that."))({ value: this.maxSelectValue })`